### ISSUE TITLE
Automated setting of created_at does not work

### ISSUE THREAD

**Issue by BenBach** [hanami/model#412]
I have a repository with a created_at column with `Null: false` constraint. My database is postgresql

When I try to do a create similar to the example code from the guide it fails because created_at can't be null (http://hanamirb.org/guides/models/repositories/)

```
repository = BookRepository.new
book = repository.create(title: "Hanami")
```

Expected: `created_at` is set automatically

**Comment by BenBach** [comment]
I found the problem:

If only a created_at column is existing this does not work. Once I add the updated_at column it works
I think having only a created at column should work as well (eg. in case of immutable objects)

**Comment by TiteiKo** [comment]
Hi @BenBach, thanks for reporting this!

I've made a quick fix, could you review it please? :)

**Comment by BenBach** [comment]
@TiteiKo I did. Thanks for fixing it so quick. :-)

**Comment by jodosha** [comment]
Fixed by #413
